spec/requests/health_check_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe 'Caching', type: :request do
```

Comment:
Think 'caching' was wrong in the original and has been copied over. 'Health check'? Shouldn't block a merge, fix if you have the time.

Reply:
Yup I'll change it